<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="txt_eather_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>271</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-left-radius:5px; border-bottom-left-radius:5px; background-color:#E8E1B1;</string>
    </property>
    <property name="text">
     <string>미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todayTendency">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>231</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-right-radius:5px; border-bottom-right-radius:5px; background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todayTemper">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>240</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#3d85c6;background-color: transparent;</string>
    </property>
    <property name="text">
     <string>12.6°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnEnter">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>12</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #333; border-radius:5px; background-color:#666; color:#fff;</string>
    </property>
    <property name="text">
     <string>날씨조회</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputArea">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>13</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom:5px solid #444; border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_W">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>-9</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#444;background-color: transparent;</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_eather_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>311</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-left-radius:5px; border-bottom-left-radius:5px; background-color:#E8D0B1;</string>
    </property>
    <property name="text">
     <string>초미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_eather_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>231</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-left-radius:5px; border-bottom-left-radius:5px; background-color:#B1D9E8;</string>
    </property>
    <property name="text">
     <string>체감온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todayDust2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>311</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-right-radius:5px; border-bottom-right-radius:5px; background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todayDust1">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>271</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-right-radius:5px; border-bottom-right-radius:5px; background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgWeather">
    <property name="geometry">
     <rect>
      <x>-260</x>
      <y>40</y>
      <width>400</width>
      <height>309</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="compYday">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>240</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>돋움</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>어제와의 날씨 비교</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todayArea">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>날씨 조회 지역 이름</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_eather">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>5</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#999;background-color: transparent;</string>
    </property>
    <property name="text">
     <string>eather</string>
    </property>
   </widget>
   <widget class="QLabel" name="todayUV">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-right-radius:5px; border-bottom-right-radius:5px; background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_eather_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-left-radius:5px; border-bottom-left-radius:5px; background-color:#E6B1C4;</string>
    </property>
    <property name="text">
     <string>자외선지수</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="todaySunTime">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>390</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-right-radius:5px; border-bottom-right-radius:5px; background-color:#fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtSun">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #999; border-top-left-radius:5px; border-bottom-left-radius:5px; background-color:#D3D3D3;</string>
    </property>
    <property name="text">
     <string>일출/일몰</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>imgWeather</zorder>
   <zorder>todayTendency</zorder>
   <zorder>bttnEnter</zorder>
   <zorder>txt_W</zorder>
   <zorder>todayDust2</zorder>
   <zorder>todayDust1</zorder>
   <zorder>compYday</zorder>
   <zorder>todayArea</zorder>
   <zorder>txt_eather</zorder>
   <zorder>inputArea</zorder>
   <zorder>todayUV</zorder>
   <zorder>todaySunTime</zorder>
   <zorder>txtSun</zorder>
   <zorder>txt_eather_5</zorder>
   <zorder>txt_eather_2</zorder>
   <zorder>txt_eather_7</zorder>
   <zorder>txt_eather_8</zorder>
   <zorder>todayTemper</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
